spec/models/describe_unknown_repair_form.rb

```diff
@@ -2,11 +2,24 @@
 
 RSpec.describe DescribeUnknownRepairForm do
   describe 'validations' do
-    it '.description' do
+    it 'is valid with a description' do
+      form = DescribeUnknownRepairForm.new(description: 'It is broken')
+
+      expect(form).to be_valid
+    end
+
+    it 'is invalid without a description' do
       form = DescribeUnknownRepairForm.new
       form.valid?
 
       expect(form.errors.details[:description]).to include(error: :blank)
     end
+
+    it 'is invalid with an overly long description' do
+      form = DescribeUnknownRepairForm.new(description: ('X' * 201))
```

Comment:
201? How come this passes?

Reply:
Good point... I will investigate :+1:

Reply:
@dgmstuart I figured out why this was passing... it wasn't.

The spec file is `spec/models/describe_unknown_repair_form.rb` is missing the `_spec` suffix, so doesn't get run. If I run the spec on it's own it fails as expected!

Reply:
I've rebased and renamed this file to `spec/models/describe_unknown_repair_form_spec.rb` and immediately saw a failing spec that I've now fixed.